### ISSUE TITLE
NetworkX's Version

### ISSUE THREAD

**Issue by xkszltl** [pytorch/pytorch#24835]
## 📚 Documentation

I wonder if this is still the case:
https://github.com/pytorch/pytorch/blob/dfdb86a59577d8b0fc4565988a8ac01b5ecd339f/docker/caffe2/jenkins/common/install_python.sh#L143-L146

I tried the latest nextworkx for memonger and haven't hit an issue so far.

**Comment by deepali-c** [comment]
The `MemongerTest.test_topological_sort_longest_path` does fail with `networkx 2.2`
Do we want to support `networkx 2.2`? I can add a PR for the same. @ezyang please advise.

**Comment by ezyang** [comment]
Yes, in general, we will always accept patches that update us to newer versions of our dependencies (fixing uses of the dependency in our code).
